release notes/v0.52.0.md

```diff
@@ -0,0 +1,141 @@
+- [`k6/browser` has graduated from an experimental module](#k6browser-has-graduated-from-an-experimental-module-3793)
+- [`k6/browser` has now a fully Async API](#k6browser-has-now-a-fully-async-api-browser428)
+
+## Breaking changes
+
+### Switch `goja` to our own fork named `sobek` [#3775](https://github.com/grafana/k6/pull/3775)
+
+To accelerate the development speed and bring [ECMAScript Modules (ESM)] support to k6 earlier (https://github.com/grafana/k6/issues/3265),
+we have decided to create a fork of the [`goja`](https://github.com/dop251/goja/) project under the Grafana GitHub organization,
+named [`sobek`](https://github.com/grafana/sobek).
+
+So, since this release, k6 (and its extensions) starts using `sobek` instead of the original `goja`, for all (of the
+publicly exposed parts of the API) except for a couple of packages that are only used internally by k6.
+
+Find further details in [#3772](https://github.com/grafana/k6/issues/3772) and [#3773](https://github.com/grafana/k6/issues/3773).
+
+### Panics are no longer being captured [#3777](https://github.com/grafana/k6/pull/3777)
+
+Since this release, Go panics are no longer being captured by k6. This means that if a panic occurs while running a test,
+the k6 process will crash, and the panic stack trace will be printed to the console.
+
+We decided to change this behavior because it's something that was left from the past, as a safeguard, but it's not as
+good as it might seem, as for most cases with multiple goroutines/async it is not enough, and also makes a bunch of potential
+bugs seem like less of an issue.
+
+Thus, this will help us to identify and fix bugs earlier, improve the overall stability of k6, and
+most likely make the experience of developing k6 extensions friendlier.
+
+### `lib.State` no longer has `Group` [#3750](https://github.com/grafana/k6/pull/3750)
+
+As the result of refactoring the implementation of `group` and `check` methods, in order to decouple them, and thus 
+enable other future improvements, the `lib.State` object no longer has a `Group` field.
+
+This change should not affect most users, except for a couple of extensions, for which the use of `Group` was
+already questionable:
+- `xk6-fasthttp`
+- `xk6-g0`
+
+### The `web-dashboard` built-in output becomes `dashboard` [xk6-dashboard#163](https://github.com/grafana/xk6-dashboard/pull/163)
+
+Since `v0.49.0`, k6 includes [xk6-dashboard](https://github.com/grafana/xk6-dashboard) as a built-in output extension.
+The name of the built-in module is the same as was the output name (`web-dashboard`) registered by the extension. 
+Therefore, if you try to use it as an extension (for example, a newer version), you will get the following error:
+
+```sh
+k6 run -o web-dashboard script.js
+
+ERRO[0000] invalid output extension web-dashboard, built-in output with the same type already exists
+```
+
+So, from now on, the registered name of the extension is again the one previously used: `dashboard`. 
```

Comment:
I found this paragraph a bit complicated from the reader's perspective; maybe we should just keep only the working example instead? Like:

If you're facing this error please, run k6 like the following:
```sh
k6 run -o dashboard script.js
```

Reply:
I also do not understand what are the implications here, why do we still care about the dashboard as an extension? And now this sounds broken. All the env vars are no longer consistent with the name.

Reply:
I'm assuming we meant here something like: from now on, use `-o dashboard`, but if you build  `xk6 build master --with "github.com/grafana/xk6-dashboard@main"` obviously the new version of dashboard will be `-o xk6-dashboard`. But I am not sure if such details are essential in release notes :shrug:

Reply:
BTW this change is irrelevant in k6 changelog, so I would remove it...

Reply:
Ok, this matches my understanding from the pull request's description. I agree with Ivan that this isn't relevant for the k6 changelog.

Reply:
Removed, as suggested! 👍🏻